consensus/src/pending_votes.rs

```diff
@@ -91,9 +91,11 @@ impl PendingVotes {
         // 1. Has the author already voted for this round?
         //
 
-        if let Some(previously_seen_vote) = self.author_to_vote.get(&vote.author()) {
+        if let Some((previously_seen_vote, previous_li_digest)) =
+            self.author_to_vote.get(&vote.author())
+        {
             // is it the same vote?
-            if li_digest == previously_seen_vote.ledger_info().hash() {
+            if &li_digest == previous_li_digest {
```

Comment:
this is unrelated to the diff, but it seems wasteful to compute hash every time here.

I can revert if this seems unnecessary.